test/plugin_functional/test_suites/core_plugins/applications.ts

```diff
@@ -91,6 +91,18 @@ export default function({ getService, getPageObjects }: PluginFunctionalProvider
       await testSubjects.existOrFail('fooAppPageA');
     });
 
+    it('navigating to chromeless application hides chrome', async () => {
+      await appsMenu.clickLink('Chromeless');
```

Comment:
May want to navigate via URL so this test doesn't break when the navlink is removed

Reply:
https://github.com/elastic/kibana/pull/50060/files#diff-03647e4cd00116d6bca6d56e69e8c5b8R99